### DISCUSSION TITLE
isManual prop does not work on theme defaultProps for Tabs

### DISCUSSION THREAD

**Discussion by mhamid3d** [chakra-ui/chakra-ui#8316] [General]
### Description

When I set `isManual: true` as a defaultProp for the tabs component theme, it does not actually set it. I tried setting it on the root style, tablist style, and through the variant, none work.

### Chakra UI Version

2.8.1

### Browser

Microsoft Edge

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
v2 theme `defaultProps` and `isManual` are gone. Use the Tabs recipe with `value` and `onValueChange`.
